powershell/llcsharp/operation/method.ts

```diff
@@ -259,13 +261,13 @@ export class OperationMethod extends Method {
         }
         yield EOL;
       }
-      yield eventListener.signal(ClientRuntime.Events.HeaderParametersAdded, urlV.value);
+      yield eventListener.signal(ClientRuntime.Events.HeaderParametersAdded);
 
       if (bp) {
         yield '// set body content';
         yield `request.Content = ${bp.serializeToContent(bp.mediaType, ClientRuntime.SerializationMode.None)};`;
         yield `request.Content.Headers.ContentType = ${System.Net.Http.Headers.MediaTypeHeaderValue.Parse(bp.contentType)};`;
-        yield eventListener.signal(ClientRuntime.Events.BodyContentSet, urlV.value);
+        yield eventListener.signal(ClientRuntime.Events.BodyContentSet);
```

Comment:
It doesn't make sense to print url when body content is set, so I simply removed it